pkg/resources/sources/options.go

```diff
@@ -3,7 +3,55 @@
 
 package sources
 
+import (
+	"fmt"
+	"strconv"
+)
+
 type Options struct {
-	Sources        []string
-	OvercommitVCPU float64
+	Sources            []string
+	OvercommitVCPU     float64
+	ReservedMemorySize MemorySize
+	BlockedHugepages   uint64
+}
+
+// MemorySize is a custom type to handle memory sizes in human-readable format.
+type MemorySize uint64
+
+func (m *MemorySize) String() string {
```

Comment:
here you can create copy (m MemorySize) pointer and uint64 have same size so you don't have any benefits with pointer.

Reply:
Done